services-api/src/main/java/io/scalecube/services/transport/api/Address.java

```diff
@@ -0,0 +1,47 @@
+package io.scalecube.services.transport.api;
```

Comment:
Move it to services-api  pzl. Let's not add additional dependencies on transport where it was absent.